docs/testing/021-API Key管理-测试.md

```
@@ -0,0 +1,166 @@
+cd backend && cargo clippy --all-targets -- -D warnings
+# 预期：零告警零错误
+```
+
+```bash
+cd frontend && npx tsc --noEmit
+# 预期：零错误
+```
+
+# 4. 测试方法
+
+### 4.1 单元测试
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)